package.json

```diff
@@ -57,7 +57,6 @@
 		"@typescript-eslint/eslint-plugin": "^4.23.0",
 		"@typescript-eslint/parser": "^4.23.0",
 		"eslint": "^7.26.0",
-		"tsc-watch": "^4.2.9",
-		"typescript": "^4.1.3"

```

Comment:
What happened here?

Reply:
this was a long while ago so i don't exactly remember, but i may have been messing around with a later version of djs. i'll get rid of this change soon